vendor/github.com/containerd/containerd/platforms/cpuinfo.go

```diff
@@ -0,0 +1,117 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package platforms
+
+import (
+	"bufio"
+	"os"
+	"runtime"
+	"strings"
+
+	"github.com/containerd/containerd/errdefs"
+	"github.com/containerd/containerd/log"
+	"github.com/pkg/errors"
+)
+
+// Present the ARM instruction set architecture, eg: v7, v8
+var cpuVariant string
+
+func init() {
```

Comment:
@jonjohnsonjr This was the init-time lookup we were trying to avoid pulling in.

Reply:
The fix in containerd is included in v1.5.0-beta.0: https://github.com/containerd/containerd/blob/v1.5.0-beta.0/platforms/cpuinfo.go

If we get moby/moby to update to that release of containerd/containerd we can pull in that version of moby/moby: https://github.com/moby/moby/blob/8d3179546e79065adefa67cc697c09d0ab137d30/vendor.conf#L133

Reply:
So something else is pulling in containerd than what we expected (this was supposed to be before the problematic commit that your fix helps with, IIUC).

Reply:
Apologies for the loooong delay in following up on this PR. I think we were mistaken that `v20.10.0-beta1` doesn't have the problematic commit, because it appears in the list of tags for https://github.com/moby/moby/commit/7a9cb29fb980c0ab3928272cdc24c7089b2fcf64. I did manage to verify that `v19.03.15` will "work" in the sense that it doesn't contain `vendor/github.com/containerd/containerd/platforms/cpuinfo.go` and also plays nicely with our repositories. I don't really understand the meaning of these versions (`v19.03.15` seems to be equivalent to `v17.12.0-ce-rc1`? At least go.mod updates to that version...) so I am not sure if updating to this version makes sense.